test-utils/derive/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "substrate-test-utils-derive"
+version = "0.8.0-rc4"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+
+[dependencies]
+quote = "1"
```

Comment:
Please put the exact version you are using.

Reply:
done